frontend/view-edit-client/client-history/client-history.component.tsx

```diff
@@ -90,9 +90,9 @@ export default function ClientHistory(props: ClientHistoryProps) {
             />
           </div>
           <div className="timeline-right">
-            <h4 className={a("title").m("outdated", log.idOfUpdatedLog)}>
+            <h2 className={a("title").m("outdated", log.idOfUpdatedLog)}>
```

Comment:
And these h3s

Reply:
Axe disagrees with you, so I'm going to stick to what it says